codec-http2/src/main/java/io/netty/handler/codec/http2/Http2CodecUtil.java

```diff
@@ -252,6 +252,14 @@ static void writeFrameHeaderInternal(ByteBuf out, int payloadLength, byte type,
         out.writeInt(streamId);
     }
 
+    static void writeFrameHeaderInternal(ByteBuf out, int payloadLength, byte type,
```

Comment:
Any reason not to define the previous implementation in terms of this new one?

Reply:
Binary compabinity

Reply:
It's not critical to change but now I'm confused. What makes this definition binary incompatible?
```
static void writeFrameHeaderInternal(ByteBuf out, int payloadLength, byte type,  Http2Flags flags, int streamId) {
        // forward params to the new method
        writeFrameHeaderInternal(out, payloadLength, type, flags.value(), streamId);
    }
```

Reply:
I see, you mean a method forwarding, that can be done, jit will inline it anyway.